sites/header/index.json-ld

```diff
@@ -30,5 +31,443 @@
+			},
+			"introduction": {
+				"en": "The difference with version 3.0 is the value for <code>col-xs-5</code> and <code>col-xs-offset-7</code> have been adjusted to a \"6\" so the button text don't brake and stays on one line for every viewports from extra-extra-small to extra-large (xxs, xxs, sm, md, lg and xl).",
+				"fr": "text... or HTML"
+			},
+			"instruction": {
+				"en": [
+					"This version requires the implementation of the <a href=\"../gcweb-menu/gcweb-menu-docs-en.html\">gcweb-menu version 2.0</a> where the <code>&lt;div class=\"container\"&gt;</code> has been relocated in the header so outside the \"gcweb-menu\" component."
+				],
+				"fr": []
+			}
+
+		}
+
+	],
+
+	"changeset": [
+		{
+			"@id": "_:cs_default",
+			"name": {
+				"en": "Version 4",
+				"fr": "Version 4"
+			},
+			"status": "stable",
+			"baseOnIteration": "_:iteration_4",
+
+			"detectableBy": "Text on how to detect",
+
+			"layout": [
+				"Row - Skip link first row centered",
+				"Row, cell - Language toggle top right",
+				"Cell - Logo brand top left",
+				"Cell - Search top right bellow language toggle",
+				"Row - Horizontal line bellow the logo brand and search box",
+				"Row, cell - Site top menu left bellow the branding",
+				"Cell - Authentication beside the top menu on its right side",
+				"Row - Breadcrumb under the top menu"
+			],
+			"style": [
+				"May have some specific style, need to be researched and centralized",
+				"Bootstrap 3 grid CSS: container, row, col-md-8, col-xs-6 col-xs-offset-6 col-md-offset-0 col-md-4 col-xs-12"
+			],
+			"semantic": {
+				"@type": "source-code",
+				"description": "Include with logic in liquid that support all variations",
+				"code": {
+					"@type": [ "@id", "rdf:HTML" ],
+					"@value": "includes/header.html"
+				}
+			},
+			"logic": "Not applicable",
+			"behaviour": "Not applicable",
+			"recognition": "The layout (disposition) of the component and the brand logo must be at top left.",
+			"guidance": "Out of scope, please see DTO related guidance on the site header component.",
+			"context": "Immediate children of the <body> element immediately after the 'skip link' component and before the <main> element",
+			"configuration":[
+				"Optional but highly recommended language toggle component",
+				"Optional site search component",
+				"Optional top menu component",
+				"Optional authentication component",
+				"Optional breadcrumbs component"
+			],
+			"static": "Not applicable, see each included component for more details",
+			"schema": "Not applicable, see each included component for more details",
+			"dependency": [
```

Comment:
```suggestion
			"dependencies": [
```

Reply:
Will stay singular as all other items are also in a singular form.